js/main.js

```diff
@@ -0,0 +1,39 @@
+let string = 'some test string';
+
+alert(string[0]); //получаем первую букву
+alert(string.length); //определяем длину
+alert(string.length-1); //последний символ строки
+alert(string[15]); //получаем последнюю буквы
+alert(string[0].toUpperCase()); //делаем первую букву большой
+alert(string[15].toUpperCase()); //делаем последнюю букву большой
+alert(string.indexOf("string")); //находим положение слова в строке
+alert(string.indexOf("string")-1); //положение второго пробела
```

Comment:
`string.indexOf(' ', string.indexOf(' ') + 1);` would be the correct way to resolve this task